StellarDotnetSdk/Responses/SorobanRpc/GetEventsResponse.cs

```diff
@@ -6,7 +6,6 @@ namespace StellarDotnetSdk.Responses.SorobanRpc;
 /// <summary>
 ///     Holds the details of the response of <c>getEvents()</c>.
 /// </summary>
-[JsonObject]
```

Comment:
Why do we remove these all of a sudden? Same counts for other files.

Reply:
The attribute is actually not necessary, I forgot to remove them in the previous PR.